R/load_flu_hosp_data.R

```diff
@@ -66,14 +71,16 @@ load_flu_hosp_data <- function(as_of = NULL,
   } else {
     locations_to_fetch <- locations
   }
-  
+
+  signal <- ifelse(pathogen=="flu", "confirmed_admissions_influenza_1d", "confirmed_admissions_covid_1d")
```

Comment:
maybe move this inside of the `if (source == "covidcast")` block since I think it's only used in there?

Reply:
yup. updating this.